docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst

```diff
@@ -44,28 +40,30 @@ Release notes: Open GEE 5.3.6
       .. rubric:: Resolved Issues
 
       .. list-table::
-         :widths: 10 30 55
+         :widths: 35 65
          :header-rows: 1
 
          * - Number
            - Description
-           - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1851

```

Comment:
It looks like we generally list issues in reverse order of issue number. Can we sort these in the same order?